.github/workflows/rotki_nightly.yml

```diff
@@ -11,44 +11,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-18.04, macos-10.15 ]
-        python-version: [ 3.9 ]
+        os: [ ubuntu-18.04, macos-10.5 ]
     runs-on: ${{ matrix.os }}
+    name: 'Backend tests on ${{ matrix.os }}'
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v3
         with:
           fetch-depth: 2
-      - name: Setup SQLCipher
-        run: |
-          if [ ${{ matrix.os }} == 'ubuntu-18.04' ];
-          then
-            sudo apt-get update
-            sudo apt-get install libxml2-utils
-            ./install_deps.sh
-            sudo ldconfig
-          fi
-          if [ ${{ matrix.os }} == 'macos-10.15' ];
-          then
-            brew install sqlcipher
-          fi
-      - name: Set up python
-        uses: actions/setup-python@v1
-        with:
-          python-version: '3.9'
-      - name: Store python Cache (linux)
-        uses: actions/cache@v2
-        if: startsWith(runner.os, 'Linux')
+      - name: Setup python
+        uses: actions/setup-python@v4
```

Comment:
Shouldn't we fix here the python version? the action allows for it

Reply:
I guess, I probably missed it